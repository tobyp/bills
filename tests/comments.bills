#!./bills
# This is a Comment

# So is the next line
#A	B	1.0

# But not this one
C	D.	1.0	# except from here on out

E F. 5#direct comment

     # whitespace comment     
