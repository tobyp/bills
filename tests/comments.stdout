todo: D pays 1.0 to C
todo: F pays 5.0 to E
